### DISCUSSION TITLE
Paste into multiple cells

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#1983] [Q&A] [answered]
Sometimes I want to copy the content of one cell and paste it into multiple adjacent cells. In Excel I can just mark those cells and press Ctrl-V. Is there a way to do that with VisiData?

**Comment by saulpw** [comment, marked as answer]
If the cells are in one column, `zy`, select the rows, then `gzp`.

**Reply by cool-RR** [reply]
Great, thank you. I didn't find that command in the help screen because it didn't have the word "paste" anywhere.
